File: ruby/getting-latest-articles/Get5ArticlesOrderedFiltered.rb
# DocSection: getting_latest_articles_get_5_articles_ord_filt
require 'delivery-sdk-ruby'

delivery_client = KenticoCloud::Delivery::DeliveryClient.new project_id: '975bf280-fd91-488c-994c-2f04416e5ee3'
delivery_client.items('system.type'.eq 'article')
               .limit(5)
               .order_by('elements.post_date', '[desc]')
               .elements(%w[title teaser_image])
               .execute do |response|
                 items = response.items
                 items.each { |item| puts item.system.codename }
               end
# EndDocSection

File: js/getting-latest-articles/Get5ArticlesOrderedFiltered.js
// DocSection: getting_latest_articles_get_5_articles_ord_filt
const KenticoCloud = require('kentico-cloud-delivery');

// Create strongly typed models according to https://developer.kenticocloud.com/docs/strongly-typed-models
class Article extends KenticoCloud.ContentItem {
    constructor() {
        super();
    }
}

const deliveryClient = new KenticoCloud.DeliveryClient({
    projectId: '975bf280-fd91-488c-994c-2f04416e5ee3',
    typeResolvers: [
        new KenticoCloud.TypeResolver('article', () => new Article())
    ]
});

deliveryClient.items()
    .type('article')
    .orderParameter('elements.post_date', KenticoCloud.SortOrder.desc)
    .limitParameter(5)
    .elementsParameter(['title', 'teaser_image'])
    .getObservable()
    .subscribe(response => console.log(response));  
// EndDocSection